src/content/ui-components/getting-started/style.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Style Tiptap UI Components
+meta:
+  title: UI Components | Tiptap
+  description: Style Tiptap UI Components to match your design system or application theme.
+  category: UI Components
+---
+
+import { Callout } from '@/components/ui/Callout'
+
+### Add Styles
+
+To enable support for SCSS, install the appropriate SCSS compiler based on your framework setup.
+
+<Callout title="Styles are only added when using the CLI" variant="info">
+  Styles are included only if you installed a component or template using the CLI. These styles come
+  bundled with whatever you add through the CLI.{' '}
+</Callout>
+
+#### For Next.js
+
+1. Install `sass`:
+
+```bash
+pnpm add -D sass
+```
+
+2. Import the SCSS partials in your main stylesheet:
+
+**File:** `global.css`
```

Comment:
Before step 2, you have to create a `global.css` file and import it in the `layout.tsx` component, right?